src/client/components/localisation/view.tsx

```diff
@@ -3,19 +3,22 @@ import { runInAction } from 'mobx'
 import { observer } from 'mobx-react'
 import * as React from 'react'
 import { WebGLRenderer } from 'three'
+import { lazyInject } from '../../../inversify.config'
 import { LocalisationModel } from './model'
 import { ViewMode } from './model'
 import { LocalisationPresenter } from './presenter'
 import * as style from './style.css'
 import { LocalisationViewModel } from './view_model'
 
 interface LocalisationViewProps {
-  model: LocalisationModel,
-  presenter: LocalisationPresenter
+  model: LocalisationModel
 }
 
 @observer
 export class LocalisationView extends React.Component<LocalisationViewProps, any> {
+  @lazyInject(LocalisationPresenter)
```

Comment:
Are we always going to lazy inject? I feel like we shouldn't know the way it's being injected in the localisation view.